sdk/go/auto/stack_test.go

```diff
@@ -252,3 +254,50 @@ func TestRefreshOptsClearPendingCreates(t *testing.T) {
 
 	assert.Contains(t, args, "--clear-pending-creates")
 }
+
+func TestPreviewImportResources(t *testing.T) {
+	t.Parallel()
+
+	// / Arrange
```

Comment:
```suggestion
	// Arrange
```

We should have a comment here explaining why we are skipping the test on windows, ideally associated with an issue to re-enable running the test there. Does this need to be disabled for windows though?  Do we already know it's not going to pass or be flaky there?

Reply:
I removed the skip, I think I copied it from another test